db/seeds.rb

```diff
@@ -0,0 +1,34 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+user_1, user_2 = User.new, User.new
+user_1.email = "lint@lint.com"
+user_2.email = "linter@linter.com"
+user_1.password = "iLoveLint"
+user_2.password = "iLoveLintToo"
+[user_1, user_2].each(&:save!)
+
+picture_1_url = "https://img.thrfun.com/img/139/690/recycling_dryer_lint_l3.jpg"
+picture_2_url = "https://cdn.instructables.com/F2S/HAGJ/F82EXR2W/F2SHAGJF82EXR2W.LARGE.jpg"
+picture_3_url = "https://i1.wp.com/krinkle.net/wordpress/img/AppleMouse/TheDirtErrLint.jpg"
+picture_4_url = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSQR_X-G9TDXsvjPixHBZCMWxTXE5CuwAoiZ05FxvmMoh64DHFv"
+
+picture_1 = Picture.create(url: picture_1_url, user_id: user_1.id)
```

Comment:
You can pass in objects rather than ids, and allow ActiveRecord to handle creating the association.

```ruby
picture_1 = Picture.create(url: picture_1_url, user: user_1)
```